Gems/EMotionFX/Code/EMotionFX/Source/Importer/ChunkProcessors.cpp

```diff
@@ -69,6 +69,7 @@
 namespace EMotionFX
 {
     AZ_CLASS_ALLOCATOR_IMPL(SharedData, ImporterAllocator, 0)
+    AZ_CLASS_ALLOCATOR_IMPL(SharedHelperData, ImporterAllocator)
```

Comment:
Should this be AZ_CLASS_ALLOCATOR_IMPL(SharedHelperData, ImporterAllocator, 0) ?

Reply:
No, in fact the third parameter to this macro can be removed altogether (it is `0` in all cases and isn't used anymore). I've updated this PR to not add the third parameter to these new uses of the macro, we can make a separate pass to remove the third parameter from the existing usages later.